test/response.test

```diff
@@ -90,15 +90,15 @@ namespace eval ::qcode::test {
 <div class="message"></div>
 <div class="record"></div>
 <div class="action"></div>
```

Comment:
Hmm guess that wouldn't work if the property's value is an array or an object ... will call to discuss in a bit anyway